src/Coordination/KeeperAsynchronousMetrics.cpp

```diff
@@ -68,6 +68,7 @@ void updateKeeperInformation(KeeperDispatcher & keeper_dispatcher, AsynchronousM
     new_values["KeeperIsFollower"] = { is_follower, "1 if ClickHouse Keeper is a follower, 0 otherwise." };
     new_values["KeeperIsObserver"] = { is_observer, "1 if ClickHouse Keeper is an observer, 0 otherwise." };
     new_values["KeeperIsStandalone"] = { is_standalone, "1 if ClickHouse Keeper is in a standalone mode, 0 otherwise." };
+    new_values["KeeperHasMemorySoftLimitHit"] = { has_mem_soft_limit_hit, "1 if ClickHouse Keeper has hit the memory soft limit, 0 otherwise." };
```

Comment:
We should clarify - "has hit" -> "is exceeding",
otherwise I read it like "has ever hit before".

Reply:
Update the nomenclature.